include/swift/Sema/ConstraintSystem.h

```diff
@@ -566,6 +566,13 @@ template <typename T = Decl> T *getAsDecl(ASTNode node) {
   return nullptr;
 }
 
+template <typename T = Decl>
```

Comment:
Just curious, IIUC wouldn't default template parameter be <typename T = Stmt>?

Reply:
Ah, that’s right!

Reply:
So much for copying… 🤦‍♂️  I’ll wait until source compact is done and amend the corresponding commit.

Reply:
Good catch, fixed!